pytorch/training/buildspec.yml

```diff
@@ -85,7 +85,7 @@ images:
   #   target: sagemaker
   #   context:
   #     <<: *TRAINING_CONTEXT
-  # BuildSageMakerGPUPTTrainPy3DockerImage:
+  # BuildSageMakerGPUPTTrainPy3cu118DockerImage:
```

Comment:
Please remove changes in pytorch/training/buildspec.yml, which is currently building only PT 2.1 DLCs.